src/libs/ReportUtils.js

```diff
@@ -2152,6 +2152,18 @@ function isReportDataReady() {
     return !_.isEmpty(allReports) && _.some(_.keys(allReports), (key) => allReports[key].reportID);
 }
 
+/**
+ * Checks if the current user is the admin of the policy.
```

Comment:
I think this would make more sense in `policyUtils` than `reportUtils`.

Reply:
I thought so, but since it was depending on report data I put it to reportUtils. All methods in policy utils are working on policy data itself. Thus I decided to do so. 

To put it in `policyUtils` we should either pass policies list or connect to onyx and get the policies. One more way, which I prefer, is to make this function work with policy data and in `ReportDetailsPage` find report's policy from policy list and pass it to this function.

Which way should I proceed?

Reply:
Yes, but the page this is being called from is the ReportDetailsPage which does have the policy as a prop. So we could call this with the policy instead of the report.

Reply:
We have the policy here: https://github.com/Expensify/App/blob/2090c0525de9cd87b8c738e9ccb7dafe79a9a229/src/pages/ReportDetailsPage.js#L96

Reply:
Fixed